metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -365,6 +404,10 @@ def build_corp_user_mce(
         if groups:
             user_snapshot.aspects.append(GroupMembershipClass(groups=groups))
 
+        self.stale_entity_removal_handler.add_entity_to_state(
+            type="corpuser", urn=f"urn:li:corpuser:{ldap_user}"
```

Comment:
please construct the `urn` using [this function](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/emitter/mce_builder.py#L170)

Reply:
will do

Reply:
completed